packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -142,7 +83,7 @@ contract LockedGold is ILockedGold, ReentrancyGuard, Initializable, UsingRegistr
    * @notice Locks gold to be used for voting.
    */
   function lock() external payable nonReentrant {
-    require(isAccount(msg.sender), "not account");
+    require(getAccounts().isAccount(msg.sender), "not account");
```

Comment:
remove string here

Reply:
Do we generally not want to provide error messages?

Reply:
I think we do, this one is not very helpful